navapbc/wic-participant-recertification-portal
# [PRP-195] Build out core table columns for staff portal

## Ticket

- https://wicmtdp.atlassian.net/browse/PRP-195
- https://wicmtdp.atlassian.net/browse/PRP-257

## Changes
> What was added, updated, or removed in this PR.

- Staff portal:
  - Build out the table and connect it to the participant database
  - Add seed data for staff porta development
  - Remove prisma as a dependency and use a separate dev container to reset and seed the database
  - Remove `curl` and use built-in `wget` instead for healthcheck
  - Add a `sqlite3` dependency override to allow development on M1 macs
  - Update favicon and logo
  - Add CSS overrides for table styles, links, and to hide menus
- Participant portal:
  - Add npm script to reset the database
  - Update names for seed local agencies and add additional local agencies for pilot
  - Add seed submission records for non-prod docker image builds

## Context for reviewers
> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers. Explain how the changes were verified.

This PR builds out the staff portal table and connects it to the participant database.

Some known issues:
- The table does not autosize correctly when the page is first loaded
- The table does not size correctly if you click on the logo and the page reloads

## Testing
> Screenshots, GIF demos, code examples or output to help show the changes working as expected. ProTip: you can drag and drop or paste images into this textbox.

To test locally, run:

1. `cd staff`
2. `docker compose up -d --build`
3. Navigate to localhost:3033
4. You should see something like the following screenshot

![Screenshot 2023-04-10 at 14-22-05 Lowdefy App](https://user-images.githubusercontent.com/67701/231615430-76cd0c24-671a-4e50-915c-efe62709691b.png)

Notes:
- Run `docker compose down` when you are done testing to cleanup your containers.
- The log out button doesn't do anything right now because AWS Cognito hasn't been connected yet
- The documents don't do anything right now because S3 hasn't been connected yet
